frontend/src/metabase/actions/containers/ActionCreator/InlineDataReference.tsx

```diff
@@ -28,7 +22,7 @@ export const DataReferenceInline = ({
   };
 
   return (
-    <DataReferenceContainer isOpen={isOpen}>
+    <DataReferenceContainer>
```

Comment:
Now this component is conditionally render in JS, so no need to use `display: none` to hide it anymore.